types/evinser.d.ts

```diff
@@ -9,208 +9,162 @@ export function prepareDB(options: any): Promise<{
             sequelize: import("sequelize").Sequelize;
             where(): object;
             getDataValue<K extends string | number | symbol>(key: K): any;
-            setDataValue<K_1 extends string | number | symbol>(key: K_1, value: any): void;
+            setDataValue<K extends string | number | symbol>(key: K, value: any): void;
```

Comment:
New typescript seems to come up with some sane typings without underscore suffixes.